bot.py

```diff
@@ -1,20 +1,49 @@
-import discord
+"""
+Author MikkMakk88, morgaesis et al.
+
+(c)2021
+"""
+
+import os
+import configparser
+import logging
+
+import discord  # pylint: disable=import-error
 
 
 class MusicBot(discord.Client):
-    def __init__(self):
-        super().__init__()
+    """
+    The main bot functionality
+    """
 
-    decorator_helper = discord.Client()
+    client = discord.Client()
```

Comment:
I would argue that "client" isn't more descriptive since we are only using the instance for it's decorators. I don't see any reason not to accept this though.

Reply:
Renamed to `_discord_helper`